styles.css

```diff
@@ -1,5 +1,7 @@
 html {
   background: #f7f7f0;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
 }
 
 body {
```

Comment:
- [ ] Please be noticed that you should make your design in the mobile size fill the full width suggested you can make the width of you body 100% and remove the position absolute.

Your design                            |  Figma design           
:------------------------------------: | :----------------------------------:
![Screenshot from 2022-05-02 21-54-13](https://user-images.githubusercontent.com/33752735/166316977-52b197b4-1be3-4c9f-8485-5617284ecbb7.png) | ![Screenshot from 2022-05-02 21-58-25](https://user-images.githubusercontent.com/33752735/166317017-e9d8536c-d2fa-408f-9cca-9769b3719677.png)

Reply:
Hey @Zelalem1222  & @Abel-desu
I'm very sorry for any inconsistency that this might cause you :pray: I know how you both as a team have worked to come this far with your portfolio project, I admire your strength and energy. I hope you find this comment appealing too.

Your contact form was not actually reviewed by the previous reviewer which is why your submission will be marked as invalid.

According to the project requirement of the [contact form milestone](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md), the majority of the requirement was not implemented which result in the form not conforming to standards as expected.
Please, try implementing all the requirement and submit for another review.

Thanks for your understanding,

Cheers and Happy Coding :computer: 



https://github.com/Zelalem1222/Personal-Portfolio/blob/258a666d9040bf6da943d19a9bb4ef8446e52fe9/index.html#L188-L193

Reply:
Hi, @zieeco Thank You for Your comment. But Can you please tell us exactly what we need to do?

Reply:
@Zelalem1222, Please, check your [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md) and follow the instructions. If you don't understand, you could ask your peers that have done it already to help you if you get stuck.